src/pymovements/gaze/io.py

```diff
@@ -33,6 +34,11 @@
 from pymovements.gaze.gaze import Gaze
 
 
+# Note: column detection for ASC files now uses simple substring matching
```

Comment:
this comment isn't probably that relevant on the module level. You can move these lines to the relevant code around lines 480 - 500.